spec/models/hydra/works/collection_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe Hydra::Works::Collection do
+  
+  describe "#save" do
+    it 'is a Hydra::PCDM::Collection' do 
```

Comment:
In hydra-pcdm this is now implemented as `Hydra::PCDM.collection? obj` and `Hydra::PCDM.object? obj`